spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/health/SpanCollectionHealthIndicator.java

```diff
@@ -0,0 +1,26 @@
+package org.springframework.cloud.sleuth.health;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+
+/**
+ * Health indicator that shows the number of accepted / dropped spans.
+ *
+ * //TODO: should this be a health indicator (it's never down) or an Endpoint ?
```

Comment:
If there were a (configurable) threshold of dropped spans (a percent), then there could be a down state, and it would be more health-like. Though I don't know if the application should necessarily be considered down because it is dropping too many spans.